src/icons.ts

```diff
@@ -61,7 +61,8 @@ const icons = {
   navigationMANAGE: imgSrc("navigationMANAGE"),
   openInAudit: imgSrc("open-in-audit"),
   paste: imgSrc("paste"),
-  pin: imgSrc("pin-icon"),
+  pinOLD: imgSrc("pin-icon"),
```

Comment:
What's the reasoning behind keeping the old one? If it's still planning on being used, can we give a more descriptive name?

Reply:
Happy to delete. I'm not entirely sure how to check if it is being used but I don't believe it is - I want to avoid breaking something haha!

Reply:
If it was being used, all that will happen is it will show the new one, as it'll reference `pin` everywhere not `pinOld`